.github/workflows/ci.yml

```diff
@@ -131,17 +136,17 @@ jobs:
       id-token: write
       pull-requests: write
     env:
-      API_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/api-perf'
+      API_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/api'
       API_TAG: ${{ matrix.sha }}
-      WEB_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/web-perf'
+      WEB_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/web'
       WEB_TAG: ${{ matrix.sha }}
-      ELIXIR_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/elixir-perf'
+      ELIXIR_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/elixir'
```

Comment:
okay so it's only the Rust images that get the debug/perf distinction?

Reply:
Yeah, for now at least. We aren't really size-sensitive on the other images since they're not distributed.